tests/bluetooth/shell/src/main.c

```diff
@@ -86,21 +91,20 @@ static void cmd_hrs(const struct shell *shell, size_t argc, char **argv)
 		return;
 	}
 
-	shell_fprintf(shell, SHELL_ERROR, "%s:%s%s\r\n", argv[0],
-		      "unknown parameter: ", argv[1]);
+	error(shell, "%s:%s%s", argv[0], "unknown parameter: ", argv[1]);
```

Comment:
to speed up code execution you could rephrase it like that:

`error(shell, "%s:%unknown parameter: %s", argv[0], argv[1]);`

Reply:
Yep, I was wondering about this when doing the initial code but then I forgot to mention why it was done like that in first place.